pam-OSC.lua

```diff
@@ -210,6 +211,20 @@ local function main()
                         '"')
             end
         end
+        
+        -- Send Timecode
```

Comment:
please add here in the Menue the Option to disable Timecode Send. Just like the Colors or the Names, so that people who don't have Timecode displayed it doesn't get spamed.